tests/unit/autocomplete/core.js

```diff
@@ -308,43 +309,51 @@ QUnit.test( "ARIA", function( assert ) {
-	assert.equal( liveRegion.children().filter( ":visible" ).text(), "java",
-		"Live region changed on keydown to announce the highlighted value" );
+		element.simulate( "keydown", { keyCode: $.ui.keyCode.DOWN } );
+		setTimeout( function() {
+			assert.equal( liveRegion.children().filter( ":visible" ).text(), "java",
+				"Live region changed on keydown to announce the highlighted value" );
 
-	element.one( "autocompletefocus", function( event ) {
-		event.preventDefault();
-	} );
-	element.simulate( "keydown", { keyCode: $.ui.keyCode.DOWN } );
-	assert.equal( liveRegion.children().filter( ":visible" ).text(), "javascript",
-		"Live region updated when default focus is prevented" );
-
-	element.autocomplete( "search", "javas" );
-	assert.equal( liveRegion.children().filter( ":visible" ).text(),
-		"1 result is available, use up and down arrow keys to navigate.",
-		"Live region for one value" );
-
-	element.autocomplete( "search", "z" );
-	assert.equal( liveRegion.children().filter( ":visible" ).text(), "No search results.",
-		"Live region for no values" );
-
-	assert.equal( liveRegion.children().length, 5,
-		"Should be five children in the live region after the above" );
-	assert.equal( liveRegion.children().filter( ":visible" ).length, 1,
-		"Only one should be still visible" );
-	assert.ok( liveRegion.children().filter( ":visible" )[ 0 ] === liveRegion.children().last()[ 0 ],
-		"The last one should be the visible one" );
-
-	element.autocomplete( "destroy" );
-	assert.equal( liveRegion.parent().length, 0,
-		"The liveRegion should be detached after destroy" );
+			element.one( "autocompletefocus", function( event ) {
+				event.preventDefault();
+			} );
+			element.simulate( "keydown", { keyCode: $.ui.keyCode.DOWN } );
+			setTimeout( function() {
+				assert.equal( liveRegion.children().filter( ":visible" ).text(), "javascript",
+					"Live region updated when default focus is prevented" );
+				element.autocomplete( "search", "javas" );
+				setTimeout( function() {
+					assert.equal( liveRegion.children().filter( ":visible" ).text(),
+						"1 result is available, use up and down arrow keys to navigate.",
+						"Live region for one value" );
+					element.autocomplete( "search", "z" );
+					setTimeout( function() {
+						assert.equal( liveRegion.children().filter( ":visible" ).text(), "No search results.",
+							"Live region for no values" );
+						assert.equal( liveRegion.children().length, 1,
+							"Should be one child in the live region after the above" );
+						assert.equal( liveRegion.children().filter( ":visible" ).length, 1,
+							"Only one should be still visible" );
+						assert.ok( liveRegion.children().filter( ":visible" )[ 0 ] === liveRegion.children().last()[ 0 ],
+							"The last one should be the visible one" );
+						element.autocomplete( "destroy" );
+						assert.equal( liveRegion.parent().length, 0,
+							"The liveRegion should be detached after destroy" );
+						ready();
+					}, 500 );
```

Comment:
This test will become very long with all these nested delays... However, if we can change the source delays to 100, maybe this can be reduced.

Reply:
Changed these to 110, tests seem to be fine with 10ms of breathing room.